context/app/static/js/components/Detail/SampleTissue/SampleTissue.jsx

```diff
@@ -1,29 +1,43 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
 import PropTypes from 'prop-types';
+
+import { LightBlueLink } from 'js/shared-styles/Links';
 import { FlexPaper } from './style';
 import SectionHeader from '../SectionHeader';
 import SectionItem from '../SectionItem';
 import SectionContainer from '../SectionContainer';
 
 function MetadataItem(props) {
-  const { label, value, ml } = props;
+  const { label, children, ml, flexBasis } = props;
   return (
-    <SectionItem label={label} ml={ml}>
-      {value || `${label} not defined`}
+    <SectionItem label={label} ml={ml} flexBasis={flexBasis}>
+      {children || `${label} not defined`}
     </SectionItem>
   );
 }
 
 function SampleTissue(props) {
-  const { mapped_organ, mapped_specimen_type, tissueLocation } = props;
+  const { mapped_organ, mapped_specimen_type } = props;
   return (
     <SectionContainer id="tissue">
       <SectionHeader>Tissue</SectionHeader>
       <FlexPaper>
-        <MetadataItem label="Organ Type" value={mapped_organ} />
-        <MetadataItem label="Specimen Type" ml={1} value={mapped_specimen_type} />
-        <MetadataItem label="Tissue Location" ml={1} value={tissueLocation} />
+        <MetadataItem label="Organ Type" flexBasis="25%">
+          {mapped_organ}
+        </MetadataItem>
+        <MetadataItem label="Specimen Type" ml={1} flexBasis="25%">
+          {mapped_specimen_type}
+        </MetadataItem>
+        <MetadataItem label="Tissue Location" ml={1}>
+          <>
```

Comment:
```suggestion
```
Is there a problem passing multiple children? I'm used to only needing to use this as a grouping at the top level.

Reply:
It affects the appearance.